transport/http/inbound.go

```diff
@@ -107,10 +126,16 @@ func (i *Inbound) start() error {
 	if i.router == nil {
 		return yarpcerrors.InternalErrorf("no router configured for transport inbound")
 	}
+	for header := range i.grabHeaders {
+		if !strings.HasPrefix(header, "x-") {
```

Comment:
woaaaaaah... are we sure we want to hardcode a dependency on "x-"?  That's an internal convention, not something that should really exist in the open source world...

Reply:
ahh, just saw your comment, carry on